src/CMakeLists.txt

```diff
@@ -10,10 +10,13 @@ if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 endif ()
 
 set(CMAKE_OSX_DEPLOYMENT_TARGET "10.9")
-set(CMAKE_OSX_SYSROOT "macosx10.12")
+set(CMAKE_OSX_SYSROOT "macosx10.13")
 set(CMAKE_CXX_STANDARD 11)
 
-execute_process(COMMAND xcrun --show-sdk-path --sdk ${CMAKE_OSX_SYSROOT} OUTPUT_VARIABLE FULL_SYSROOT_PATH OUTPUT_STRIP_TRAILING_WHITESPACE)
+# use CMAKE_OSX_SYSROOT as is if it points to existing directory, lookup full path otherwise
+if(NOT EXISTS CMAKE_OSX_SYSROOT)
+  execute_process(COMMAND xcrun --show-sdk-path --sdk ${CMAKE_OSX_SYSROOT} OUTPUT_VARIABLE CMAKE_OSX_SYSROOT OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
It should be possible to fail the whole thing if `xcrun` fails. I suppose `RESULT_VARIABLE` must be passed and checked thereafter.